http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/message/HTTPCarbonMessage.java

```diff
@@ -53,8 +54,9 @@
 
     private MessagingException messagingException = null;
     private MessageDataSource messageDataSource;
-    private ServerConnectorFuture serverConnectorFuture = new HttpWsServerConnectorFuture();
     private MessageFuture messageFuture;
+    private final ServerConnectorFuture serverConnectorFuture = new HttpWsServerConnectorFuture();
+    private final HttpResponseStatusFuture httpResponseStatusFuture = new HttpResponseStatusFuture();
```

Comment:
Shall we say httpOutboundRespFuture and httpOutboundRespStatusFuture

Reply:
ack